UniMath/README.md

```diff
@@ -37,6 +37,46 @@ accidentally commit and push, add its name to the head of the list in
 "../build/Makefile-configuration", which is created from
 "../build/Makefile-configuration-template".
 
+## The UniMath formal language
+
+The formal language used in the UniMath project is based on Martin-Löf type
+theory, as present in MLTT79 below.  We are currently on version 2.
+
+UniMath-1 is MLTT79 except:
+
+- the bound variable in a λ-expression is annotated with its type
```

Comment:
With the newer 1998 version of MLTT the bound variables in λ-expressions are already tagged, and there is no reflection for identity types.

Reply:
For having bound variables explicitly typed, see section 2.3.3 in Martin-Löf's paper: "If `x` is a variable of type `A` and `b[x]` is a term of type `B[x]` then `(λ x) b[x]` is a term of type `∏(x ∈ A) B[x]". I read this as having `x` explicitly typed. In contrast, earlier versions of MLTT (say in 1984 paper) say that `λ x . x` is a term which has many types.

Reply:
You're referring to the 1998 paper.   I read "x is a variable of type A" as a statement of something appended to the context in a prerequisite of an inference rule, and I read `(λ x) b[x]` as a description of the term -- with no `A` in it.

Reply:
One advantage of referring to the older paper is that it gives credit to Per for developing the ideas by that early date.  (significantly earlier)

What if I were to refer to the later paper, not as part of our definition of the UniMath language, but as an alternative exposition?